internal/armada/server/submit_test.go

```diff
@@ -469,38 +469,6 @@ func TestSubmitServer_SubmitJob_ReturnsJobItemsInTheSameOrderTheyWereSubmitted(t
 	})
 }
 
-func TestSubmitServer_SubmitJobs_HandlesDoubleSubmit(t *testing.T) {
```

Comment:
This check has been removed because this is tersting deduplicationlogic on the old submitserver.  We've now removed this logic because we don't need dedupe logic in two places.